docs/rfc/block-registration.md

```diff
@@ -149,11 +149,11 @@ Blocks are grouped into categories to help users browse and discover them.
 
 The core provided categories are:
 
--   common
--   formatting
--   layout
+-   text
+-   media
+-   design
 -   widgets
--   embed
+-   embeds
```

Comment:
```suggestion
-   embed
```

`embed` is correct, see:
https://github.com/WordPress/wordpress-develop/blob/24465974cd6333137490b55ab663f3d1f844a95e/src/wp-admin/includes/post.php#L2180-L2211